strip-comments.js

```diff
@@ -173,4 +183,4 @@ var CommentStripper = (function (window) {
 
     return window.CommentStripper = CommentStripper;
 
-})(window);
\ No newline at end of file
+})(typeof window == "undefined" ? global : window);
```

Comment:
ah i see you mentioned that above, gotcha, and sure go ahead (just test for existence of `exports` first so it will still work in browser as vanilla js)